test/commands/extended/get_inputs_test.py

```diff
@@ -926,11 +947,8 @@ def test_security_level_1_no_stop(self):
     seed = Seed.random()
     address = AddressGenerator(seed, security_level=1).get_addresses(0)[0]
 
-    self.adapter.seed_response('getBalances', {
-      'balances': [86],
-    })
-    # ``getInputs`` uses ``findTransactions`` to identify unused
-    # addresses.
+    # ``getInputs`` uses ``findTransactions`` and
+    # ``wereAddressesSpentFrom`` to identify unused addresses.
```

Comment:
❔ We didn't modify `get_inputs.py` in this PR — is this comment accurate?

Reply:
`get_inputs` uses `iter_used_addresses`, which was modified according to this comment.